rust/linpack/Linpack.rs

```diff
@@ -0,0 +1,27 @@
+use std::time::Instant;
+
+const ARRAY_SIZE: usize = 2000;
+
+struct LinpackResult {
+    norma: f64,
+    residual: f64,
+    residn_result: f64,
+    eps_result: f64,
+    time_result: f64,
+    mflops_result: f64,
+}
+
+fn second() -> f64 {
+    let start = Instant::now();
+    start.elapsed().as_secs_f64()
+}
+
+// Implement matgen, lran, dgefa, dgesl, daxpy, ddot, dscal, idamax, epslon, dmxpy functions here
```

Comment:
Implement those functions. See go/linpack/Linpack.go, javascript/linpack/Linpack.js, php/linpack/Linpack.php, go/linpack/Linpack.go for algorithm.